src/devices/ShiftRegister/samples/Program.cs

```diff
@@ -12,7 +12,7 @@
 // Uncomment this code to use SPI (and comment the line above)
 // SpiConnectionSettings settings = new(0, 0);
 // using var spiDevice = SpiDevice.Create(settings);
-// var sr = new Sn74hc595(spiDevice, Sn74hc595.PinMapping.Standard);
+// using ShiftRegister sr = new(spiDevice, 8);
```

Comment:
why this change? Isn't example with specific shift register better?

Reply:
This line was giving error because `src/devices/ShiftRegister/samples/ShiftRegister-driver.csproj` project has no reference to the `src/devices/Sn74hc595/Sn74hc595.csproj` project and also `Sn74hc595.PinMapping.Standard` is no longer a valid way to pass pin mapping because it was refactored.

So I changed it to be similar to this line: https://github.com/dotnet/iot/blob/c8ea79c62cfbde7bad5c24ee598e9402ea3ff579/src/devices/ShiftRegister/samples/Program.cs#L10
I guess ShiftRegister samples should use the generic `ShiftRegister` class and Sn74hc595 samples should use the specific `Sn74hc595` class.